src/VisualStudio/Core/Def/Implementation/ProjectSystem/RunningDocumentTableEventTracker.cs

```diff
@@ -25,24 +28,27 @@ internal sealed class RunningDocumentTableEventTracker : IVsRunningDocTableEvent
         private readonly ForegroundThreadAffinitizedObject _foregroundAffinitization;
         private readonly IVsEditorAdaptersFactoryService _editorAdaptersFactoryService;
         private readonly IVsRunningDocumentTable4 _runningDocumentTable;
-        private readonly IRunningDocumentTableEventListener _listener;
+        private readonly IRunningDocumentTableEventListener? _listener;
         private uint _runningDocumentTableEventsCookie;
 
         public RunningDocumentTableEventTracker(IThreadingContext threadingContext, IVsEditorAdaptersFactoryService editorAdaptersFactoryService, IVsRunningDocumentTable runningDocumentTable,
```

Comment:
It feels a bit odd that the event tracker doesn't actually do events necessarily anymore. Should we instead just be moving TryGetBufferFromMoniker to an extension method on IVsRunningDocumentTable?